### PR TITLE
Re-enable building symbols for runtime store

### PR DESCRIPTION
**Pull request by JunTaoLuo**
By adding Microsoft.DiaSymReader.Native.*.dll to the path

Addresses #135 

Blocked on errors:
```
    Error generating PDB for 'C:\gh\MetaPackages2\src\Microsoft.AspNetCore.RuntimeStore\bin\work\Build.RuntimeStore.References_2.0.0-preview2--\runtimopt\Microsoft.Extensions.Hosting.Abstractions.dll': Unspecified error (Exception from HRESULT: 0x80004005 (E_FAIL))
EXEC : error : compilation failed for "C:\gh\MetaPackages2\src\Microsoft.AspNetCore.RuntimeStore\bin\work\Build.RuntimeStore.References_2.0.0-preview2--\runtimopt\Microsoft.Extensions.Hosting.Abstractions.dll" (0x80004005) [C:\gh\MetaPackages2\src\Microsoft.AspNetCore.RuntimeStore\bin\work\Restore.csproj] [C:\gh\MetaPackages2\.build\KoreBuild.proj]
C:\Users\johluo\.dotnet\x64\sdk\2.0.0-preview2-006215\Sdks\Microsoft.NET.Sdk\build\Microsoft.NET.CrossGen.targets(162,5): error MSB3073: The command "C:\Users\johluo\.nuget\packages\runtime.win-x64.microsoft.netcore.app\2.0.0-preview2-25401-01\tools\crossgen.exe -readytorun -platform_assemblies_paths C:\gh\MetaPackages2\src\Microsoft.AspNetCore.RuntimeStore\bin\work\Build.RuntimeStore.References_2.0.0-preview2--\runtimeref;C:\gh\MetaPackages2\src\Microsoft.AspNetCore.RuntimeStore\bin\work\Optimize\netcoreapp -CreatePDB C:\gh\MetaPackages2\src\Microsoft.AspNetCore.RuntimeStore\bin\work\Build.RuntimeStore.References_2.0.0-preview2--\runtimesymbols\microsoft.extensions.hosting.abstractions\2.0.0-preview2-25358\lib\netstandard2.0 C:\gh\MetaPackages2\src\Microsoft.AspNetCore.RuntimeStore\bin\work\Build.RuntimeStore.References_2.0.0-preview2--\runtimopt\Microsoft.Extensions.Hosting.Abstractions.dll" exited with code -2147467259. [C:\gh\MetaPackages2\src\Microsoft.AspNetCore.RuntimeStore\bin\work\Restore.csproj] [C:\gh\MetaPackages2\.build\KoreBuild.proj]
```

### CODE REVIEW

File: build/repo.targets [hunk 1]
```diff
@@ -130,6 +130,19 @@
     <WriteLinesToFile File="$(ArtifactsDir)version.txt" Lines="$(VersionPrefix)-$(VersionSuffix)" Overwrite="true" Condition="'$(OSPlatform)'=='Linux'" />
   </Target>
 
+  <Target Name="AddDiaSymReaderToPath" Condition="'$(OS)' == 'Windows_NT'">
+    <!-- Parse framework version -->
+    <GetDotNetHost>
+      <Output TaskParameter="DotNetDirectory" PropertyName="DotnetHomeDirectory" />
+    </GetDotNetHost>
+
+    <Exec Command="powershell.exe -command &quot;dotnet --info | Select-String version | Select-Object -Last 1 | %% {$_.ToString().SubString($_.ToString().IndexOf(':') + 1).Trim()}&quot;" ConsoleToMSBuild="true">
```
Comment (JunTaoLuo) [inline build/repo.targets]: ☹️

**Comment by ryanbrandenburg** [comment]
Is this the right fix? I don't know symbol generation very well, but I would have guessed that it was a fairly common scenario which had a simpler implementation than this. Do we have an example of another area doing a similar thing?

**Comment by JunTaoLuo** [comment]
@ryanbrandenburg This is not the normal symbol generation, it's the symbol generation for runtime store binaries which were cross-gen'ed. This is different from the symbol generation for our normal packages.

**Comment by JunTaoLuo** [comment]
Blocked on https://github.com/dotnet/sdk/issues/1302.

**Comment by JunTaoLuo** [comment]
:up::date: no longer blocked since the dotnet store issue turned out to be a long path issue.

**Comment by JunTaoLuo** [comment]
Will also need to update the CI configurations to flow the .tar.gz files

**Comment by JunTaoLuo** [comment]
ping @natemcmaster @ryanbrandenburg @pranavkm

**Comment by natemcmaster** [comment]
I already signed off

**Comment by JunTaoLuo** [comment]
@natemcmaster there were a few more changes after you initially signed off so I thought you might want to take another look.

File: tools/zip2tgz.sh [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    echo "Cleaning up"
+    rm -rf $tmp
+    echo "Done"
+}
+trap cleanup INT TERM EXIT
+set -e
+
+unzip -q $src -d $tmp
+chmod -R +r $tmp
+tar -c -z -f $dest -C $tmp .
```
Comment (natemcmaster) [inline tools/zip2tgz.sh]: Nit: trailing newline

**Pull request merged**